jsapi/test/api.test.ts

```diff
@@ -82,6 +82,50 @@ describe('inspectAddress', () => {
         message: "Bech32 error: Invalid character(s) in string"
       });
     });
+    it('Typo in Shelley', () => {
+      const addr = "addr1q9rad75p5r9qk9pnczhyqhkqfv5j2hp7lrmwxpusfqmc8pp7y5xc6cr99fsjcw0hqksgwv3enatmz9ulpp6zn2kuskaqmq72h5";
+      return expect(inspectAddress(addr)).rejects.toEqual({
+        error: { code: "decode" },
+        message: "Bech32 error: Invalid character(s) in string:\naddr1q9rad75p5r9qk9pnczhyqhkqfv5j2hp7lrmwxpusfqmc8pp7y5xc6cr99fsjcw0hqksgwv3enatmz9ulpp6zn2kuskaqmq72h5"
```

Comment:
Intresting that CLI highlights the character that is wrong which cannot be achieved in the TS binding I guess :shrug: 

![Screenshot from 2021-06-23 17-32-58](https://user-images.githubusercontent.com/42900201/123125746-15e59280-d449-11eb-9366-a94f6441e04e.png)

Reply:
Yeah, this would be useful, and is definitely possible.
I believe it's just that the full error information is thrown away in the `instance ToJSON ErrInspectAddress` that I wrote.